lib/Events/BotDisabledEvent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Events;
+
+use OCA\Talk\Model\BotServer;
+use OCA\Talk\Room;
+
+class BotDisabledEvent extends ARoomEvent {
```

Comment:
Should be documented in https://github.com/nextcloud/spreed/blob/cc4e29258c824e59fe8116fad391ec07070487dc/docs/events.md#other-events

Reply:
> Should be documented in
It will be released with next major so Since: will be 20.0.0 right?

Reply:
Yeah